scripts/micro_benchmarks.py

```diff
@@ -70,13 +70,43 @@ def __get_framework_channel_map() -> dict:
         }
 
     @staticmethod
-    def get_channel(framework: str) -> str:
+    def __get_framework_target_framework_moniker_map() -> dict:
```

Comment:
nit: rename function to something more meaningful (`__get_corert_target_framework_mapping`)

Reply:
I decided to remove it and replace entire method with one if statement.